dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -196,7 +196,7 @@
 
     @JsonIgnore
     @JoinColumn(name = "checkerid", unique = true)
-    @OneToOne(targetEntity = BioWorkflow.class, fetch = FetchType.EAGER)
+    @ManyToOne(targetEntity = BioWorkflow.class, fetch = FetchType.LAZY)
```

Comment:
Only mentioning this because we experimented with it at our 1-on-1 for the slow query problem; this didn't fix my issue, it only postponed it.

Before: Fetch the workflow, which fetches the (slow to respond) checker workflow
After applying the change: Fetches the workflow, which doesn't fetch the checker workflow. But when you navigate to the checker workflow, it's still slow to fetch.

Reply:
Makes sense, in your case I guess you're making it fetch the checker workflow.